db/migrations/public/20231127214320_create-charge-versions-view.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const viewName = 'charge_versions'
+
+exports.up = function (knex) {
+  return knex
+    .schema
+    .createView(viewName, (view) => {
+      // NOTE: We have commented out unused columns from the source table
+      view.as(knex('charge_versions').withSchema('water').select([
+        'charge_version_id AS id',
+        'licence_ref',
+        'scheme',
+        'external_id',
+        'version_number',
+        'start_date',
+        'status',
+        'apportionment',
+        'error',
+        'end_date',
+        'billed_upto_date',

```

Comment:
Upto isn't a word so we can correct this in the view.

```suggestion
        'billed_upto_date AS billed_up_to_date',

```